health/health.d/wmi.conf

```diff
@@ -6,7 +6,7 @@
     class: Utilization
      type: Windows
 component: CPU
-       os: linux
+       os: *
```

Comment:
What other OS do you expect?

Reply:
When I used FreeBSD to monitor a Windows I could have alerts.